packages/studio-web/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
Why is this file here? workflows needs to be from the root of the repo in the `.github` directory, there, no?